packages/pharos/src/components/checkbox-group/pharos-checkbox-group.wc.stories.mdx

```diff
@@ -4,20 +4,19 @@ import { action } from '@storybook/addon-actions';
 
 import './pharos-checkbox-group';
 import '../checkbox/pharos-checkbox';
+import '../link/pharos-link';
 import createFormData from '../../utils/createFormData';
 
-import Intro from '@docs/guidelines/checkbox.docs.mdx';
-
-<Intro />
-
 <Meta
   title="Forms/Checkbox Group"
   parameters={{
     options: { selectedPanel: 'addon-controls' },
   }}
 />
 
-# Examples
+# Checkbox Group
+
+<pharos-link href="https://www.jstor.org/pharos/components/checkbox" target="_blank">See guidelines in Pharos</pharos-link>
```

Comment:
Future improvement might be to find a way to include this section in all stories in a data-driven way such that we don't need to remember to do it, and won't get the link incorrect.